packages/app/src/cli/services/flow/utils.ts

```diff
@@ -0,0 +1,21 @@
+import {joinPath} from '@shopify/cli-kit/node/path'
+import {glob, readFile} from '@shopify/cli-kit/node/fs'
+
+/**
+ * Loads the schema from the partner defined file.
+ */
+export const loadSchemaPatchFromPath = async (extensionPath: string, patchPath: string | undefined) => {
+  if (!patchPath) {
+    return ''
+  }
+
+  const path = await glob(joinPath(extensionPath, patchPath))
+
+  if (path.length > 1) {
+    throw new Error('Multiple files found for schema patch path')
```

Comment:
The field was originally called `schema_patch` but we changed that halfway through the project and never updated the error message which might now be confusing to users. I think this can say "Multiple files found for schema path" and the next error "No file found for schema path".

Same for the argument and the name of the function. We can probably get rid of the "patch" part.